comms/lighthouse/src/handlers.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
What does this do?

Reply:
This comments avoids `eslint` to fail when a value is cast to object